snp_motion_planning/src/planning_server.cpp

```diff
@@ -398,6 +398,15 @@ class PlanningServer
 
       // Create a manipulator info and program from the service request
       const std::string& base_frame = req->tool_paths.paths.at(0).segments.at(0).header.frame_id;
+      if(base_frame.empty()){
+        RCLCPP_WARN(node_->get_logger(), "Base frame is empty!");
+      }
+      if(req->motion_group.empty()){
+        RCLCPP_WARN(node_->get_logger(), "Motion group is empty!");
+      }
+      if(req->tcp_frame.empty()){
+        RCLCPP_WARN(node_->get_logger(), "TCP frame is empty!");
+      }
```

Comment:
Rather than just writing out a warning this should also return a failed motion plan.

Reply:
Throw exceptions here instead so the catch block will set the response to false and propagate the message